frontend/src/components/VAudioThumbnail/VAudioThumbnail.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <!-- Should be wrapped by a fixed-width parent -->
   <div
-    class="relative h-0 w-full overflow-hidden bg-yellow pt-full"
+    class="relative h-0 w-full overflow-hidden bg-bg-fill-complementary pt-full"
```

Comment:
I feel very weird about names like `text-text` and `bg-bg-fill-complementary`.  If some colours are specifically meant for specific usage, should we consider setting them in `theme.textColor` for text colours (and `theme.backgroundColor` for background colours) instead of the general `theme.colors`? We'll be able to avoid the `text-`/`bg-` prefix which leads to the repetition.

Reply:
@dhruvkb the textColor and backgroundColor keys solution is brilliant. I'll push that shortly. I've renamed the `bg-bg`, `text-text`, and so on to use `-base` instead.